rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -148,6 +160,13 @@ service Admin {
 void ping()
 
 /*
+ * Add the DB to the host, throw exception if it already exists
+ * and overwrite is false.
```

Comment:
sure. This way the pattern can be
1) AddDB (I will add in some clearing)
2) Download from s3

Versus
1) Download from s3
2) Catch exception
3) AddDB 
4) Download from s3